lib/fruits.js

```diff
@@ -26,7 +26,7 @@ Fruit.prototype.fall = function() {
 };
 
 Fruit.prototype.collectible = function() {
-  return (this.y > this.startingY + 10) && (this.status !== "collected");
+  return ((this.y > this.startingY + 10) || (this.y > this.canvas.height - this.height - 30)) && (this.status !== "collected");
 };
 
```

Comment:
I think this should handle the issue with randomly not being able to collect the fruit.